### PR TITLE
ONE gen black VAR created plus brands carousel

### PR DESCRIPTION
**Pull request by Ksz-K**
To avoid creation of third VAR that equals #2a2a2a the gen-black=#2a2a2a was set up and replaced old black VARs (footerClaimBg i textColor)

Carousel of brands pretends to be Multi Items carousel but it is NOT as js script (jQuery prefered)  is required to allow movement of single item. Bootstrap does not support natively multi Item carousel. Trick used is not documented and bases on multi images with responsive width custom parameter.

*jQuery is not allowed in the Project - js script was not allowed in this task.

### CODE REVIEW

File: src/sass/components/_brands-carousel.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  width: 22.2%;
+  @media (min-width: 576px) {
+    width: 18%;
+  }
+  @media (min-width: 1200px) {
+    width: 16.2%;
+  }
+}
+
+.px-6 {
```
Comment (Neonides) [inline src/sass/components/_brands-carousel.scss]: Ta nazwa klasy jest chyba błędna?

Comment (Ksz-K) [reply src/sass/components/_brands-carousel.scss]: Bootstrap defined padding horizontal classes .px-# from #0 to #5 (0,4,8,16,24,48 - px respectively)
I added class px-6 (custom - not Bootstrap documented) as 100px - it is included in _brands-carousel.scss file.

Comment (Neonides) [reply src/sass/components/_brands-carousel.scss]: No to w takim razie, nie jest to miejsce na tą klasę. Skoro ma być używana przez wszystkich nie powinna być w pliku odpowiedzialnym za karuzelę, tylko w pliku gdzie są klasy ogólno dostępne

Comment (Ksz-K) [reply src/sass/components/_brands-carousel.scss]: done

**Comment by matwjablonski** [comment]
please resolve conflicts

**Comment by Ksz-K** [comment]
What it is the next step? I have resolved conflicts and pressed merged button.
All checks have failed. What I am supposed to do now?

**Pull request merged**